sql/core/src/test/scala/org/apache/spark/sql/sources/InsertSuite.scala

```diff
@@ -1043,9 +1043,8 @@ class InsertSuite extends DataSourceTest with SharedSparkSession {
 
   test("SPARK-38336 INSERT INTO statements with tables with default columns: negative tests") {
     object Errors {
-      val COMMON_SUBSTRING = " has a DEFAULT value"
+      val COMMON_SUBSTRING = "has an invalid DEFAULT value"
```

Comment:
TODO: we should move default value tests to a new suite and use the new error framework to check errors.